acceptance-tests/src/test/kotlin/com/chutneytesting/acceptance/tests/actions/HttpServerTaskAcceptanceTest.kt

```diff
@@ -0,0 +1,78 @@
+/*
+ * SPDX-FileCopyrightText: 2017-2024 Enedis
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+package com.chutneytesting.acceptance.tests.actions
+
+import com.chutneytesting.acceptance.common.checkScenarioSuccess
+import com.chutneytesting.acceptance.common.createEnvironment
+import com.chutneytesting.acceptance.common.createScenario
+import com.chutneytesting.acceptance.common.executeScenario
+import com.chutneytesting.kotlin.dsl.Scenario
+
+
+val `Http post request local server endpoint` = Scenario(title = "Http post request local server endpoint") {
+  Given("a target for the http server mock") {
+    createEnvironment(
+      "HTTP_SERVER_ENV",
+      """
+        [
+            {
+                "name": "test_http",
+                "url": "https://localhost:11789",
```

Comment:
Hardcoded port.
Maybe use our spEL function `freeSystemPort`.

Reply:
it's hard to generate a free port on docker machine and retrieve it to create the target